app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,74 @@
+module API
+  class ItemsController < ApplicationController
+    before_filter :validate_item, only: [:create, :update]
```

Comment:
@mjgiarlo have you tested posting to the API from the 'outside'? The Rails  test environment sets `config.action_controller.allow_forgery_protection = false`, but in production this defaults to true if I am not mistaken.